portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,42 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>ATP's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>ATP's Portfolio</h1>
+        <p>Welcome~This is ATP!</p>
+      
+      <h2>Who Am I?</h2>
+        <p>My name is Yuxuan Zhang, a student from Peking University, majoring CS.</p>
+        <p>"ATP" is my nickname. </p>
+        <p>It is the ID I used when I was preparing for the Olympiad in Informatics.</p>
+        <p>Thanks to my OI exprience, I'm a skilled coder. But I'm VERY poor in math!</p>
+        <p>So my motto is: <em>It looks right!</em> (Because I can't prove it :P )</p>
+        <p>Click <a href="aboutme.html">here</a> to learn more about me</p>
+      
+      <h2>My Gallery</h2>
+        <p>This is me! ↓ </p>
+        <img src="image/me.jpg" width="400"/>
+        <p>These are me and my boyfriend Aaron! (⺣◡⺣)♡ ↓ </p>
+        <img src="image/two2.png" width="600"/>
+        <p>This is my sister's cat, Yuri. How beautiful she is! ↓</p>
+        <img src="image/yuri.png" width="400"/>
+        <p>This is my dog, Duoduo. She's ten years old, an old lady(XD) ↓ </p>
+        <img src="image/duoduo.png" width="400"/>
+        <p>Click <a href="gallery.html">here</a> to see more!</p>
+
+      <h2>Let's make friends!</h2>
+        <p>E-mail: zhangyx00@pku.edu.cn</p>
+        <p>Wechat: FromATP</p>
+        <p>Steam Friend Code: 919051252 (XD </p>
+
+<!--    <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>-->
```

Comment:
Pls remove

Reply:
![tmp](https://user-images.githubusercontent.com/47445234/87238787-a5ad4f00-c439-11ea-81ce-82e566a4a1fe.png)

I have deployed it on Google Cloud Platform, on http://yzhang2-sps-summer20.appspot.com/

Reply:
Redundant comments have been removed.

Reply:
You can mark the conversation as resolved once you believe no more action is needed from you side.